filebeat/module/pensando/dfw/config/dfw.yml

```
@@ -0,0 +1,23 @@
+{{ if eq .input "syslog" }}
+
+type: udp
+udp:
+host: "{{.syslog_host}}:{{.syslog_port}}"
+
+{{ else if eq .input "file" }}
+
+type: log
+paths:
+{{ range $i, $path := .paths }}
+  - {{$path}}
+{{ end }}
+exclude_files: [".gz$"]
+
+{{ end }}
+
+processors:
+  - add_locale: ~
+  - add_fields:
+    target: ''
+    fields:
+      ecs.version: 1.7.0
```

```suggestion
  - add_fields:
      target: ''
      fields:
        ecs.version: 1.7.0
```

You need to indent this, seems to be making the tests fail 

`ERROR	instance/beat.go:952	Exiting: Failed to start crawler: starting input failed: Error while initializing input: each processor must have exactly one action, but found 3 actions (fields,add_fields,target)`